phoenix-spark-base/src/main/java/org/apache/phoenix/spark/datasource/v2/reader/PhoenixDataSourceReader.java

```diff
@@ -181,10 +184,12 @@ public List<InputPartition<InternalRow>> planInputPartitions() {
 
                 // Get the region size
                 long regionSize = CompatUtil.getSize(regionLocator, connection.getAdmin(), location);
-
+                byte[] tableBytes = PTableImpl.toProto(queryPlan.getTableRef().getTable()).
```

Comment:
Can we have a bit more descripte name ?
Like pTableCacheBytes ?

Reply:
done.